tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/observability/tracing/MainFunctionTestCase.java

```diff
@@ -53,9 +53,8 @@ public void testMainMethod() throws Exception {
         final String entryPointFunctionModule = DEFAULT_MODULE_ID;
         final String entryPointFunctionName = "main";
         final List<BMockSpan.CheckPoint> expectedCheckpoints = Arrays.asList(
-                new BMockSpan.CheckPoint(entryPointFunctionModule, FILE_NAME + ":20:5"),
                 new BMockSpan.CheckPoint(entryPointFunctionModule, FILE_NAME + ":22:13"),
-                new BMockSpan.CheckPoint(entryPointFunctionModule, FILE_NAME + ":25:23"),
+                new BMockSpan.CheckPoint(entryPointFunctionModule, FILE_NAME + ":24:15"),
```

Comment:
@nadundesilva please check

Reply:
The checkpoints are used for recording the control flow. The change should be ok since the control flow doesn't change between these positions that changed.